docs/index-server.md

```diff
@@ -21,7 +21,7 @@
 
 Carbon currently prunes and caches all block/blocklet datamap index information into the driver for
```

Comment:
Remove datamap keyword whereever mentioned

Reply:
These changes are handled in PR3738